test/e2e/cli_test.go

```diff
@@ -37,9 +37,16 @@ func (s *CLISuite) TestCompletion() {
 func (s *CLISuite) TestToken() {
 	s.Given().RunCli([]string{"token"}, func(t *testing.T, output string, err error) {
 		assert.NoError(t, err)
-		token, err := s.GetServiceAccountToken()
-		assert.NoError(t, err)
-		assert.Equal(t, token, output)
+		var authString, token string
+		token = s.GetBasicAuthToken()
+		if token == "" {
```

Comment:
Why is this if/else needed?

Reply:
This is mainly for running `test-e2e` on local.  `k3d` is using `basic auth`  and `minio` is using serviceaccount token.  This `if` will detect the auth scheme in runtime.